docs/methodology.md

```diff
@@ -52,515 +56,111 @@ The dataset is a 258 GiB subset of a larger [14 TiB][hadoop-14TB] dataset (speci
 1-3).
 
 The semi-structured log dataset, [mongodb], contains log events generated by MongoDB when running
-YCSB workloads A-E repeatedly. The dataset is 64.8 GiB and contains 186,287,600 log events.
+YCSB workloads A-E repeatedly. The dataset is 64.8 GiB and contains 186,287,600 log events. We
+concatenate the logs into a single file for benchmarking.
```

Comment:
Why?

Reply:
lol can't actually remember what I was thinking, I would say this is a decision made at that time. Does this matter? I think it makes no difference to the current result (a bit of file IO overhead probably, but should be niche)

Reply:
Sure. Was just curious.